Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: build install run test uninstall deploy manifest fmt vet generate docker-build release
 # Image URL to use all building/pushing image targets
-DEFAULT_IMG = gcr.io/engineering-devops/ds-operator
+DEFAULT_IMG = gcr.io/forgeops-public/ds-operator
```

Comment:
I assume this will be accompanied by a cloudbuild change.

Reply:
This is already done